tests/test_flask_api.py

```diff
@@ -0,0 +1,75 @@
+import json
+import pytest
+
+import flask_api
+from database_wrapper import (NimbusMySQLAlchemy, BadDictionaryKeyError, BadDictionaryValueError,
+                              NimbusDatabaseError, UnsupportedDatabaseError, BadConfigFileError)
+from mock import patch, Mock
+from .MockEntity import MockEntity
+
+
+BAD_REQUEST = 400
+SUCCESS = 200
+SERVER_ERROR = 500
+TOKEN = "test_token"
+
+
+@pytest.fixture
+def client():
+    flask_api.app.config['TESTING'] = True
+
+    with flask_api.app.test_client() as client:
+        yield client
+
+
+def test_hello(client):
```

Comment:
Not a particularly helpful test but I figured I'd add it so other tests can build off this one

Reply:
This does test for a basic problem with flask starting, I think its reasonable.